src/main/java/bio/terra/workspace/db/WorkspaceStageEnum.java

```diff
@@ -0,0 +1,58 @@
+package bio.terra.workspace.db;
+
+import bio.terra.workspace.db.exception.InvalidWorkspaceStageException;
+import bio.terra.workspace.generated.model.WorkspaceStageEnumModel;
+
+/**
+ * Internal representation of the workspace stage used for gating MC Terra features from Rawls
+ * workspaces. This is the enum serialized into the Workspace db table, not the enum generated as
+ * part of the external API. Current values RAWLS_WORKSPACE or MC_WORKSPACE.
+ */
+public enum WorkspaceStageEnum {
```

Comment:
maybe `WorkspaceStage`? Don't think we need hungarian notation.

Reply:
s.g. A few other enums from Swagger do the same, though there's no strong reason for it